crates/bevy_ui/src/ui_node.rs

```diff
@@ -410,3 +410,24 @@ pub struct CalculatedClip {
     /// The rect of the clip
     pub clip: Rect,
 }
+
+/// Indicates that this node has special requirements for the order of depth in which
+/// it should appear in the UI instead of relying solely on the hierarchy of the UI.
+///
+/// Omitting this component on a node will yield the same result as using [`ZIndex::Local(0)`].
+#[derive(Component, Copy, Clone, Debug, Reflect)]
+pub enum ZIndex {
+    /// Indicates the order in which this node should be rendered relative to its siblings.
+    ///
+    /// For root UI nodes (nodes that have to parent), using this and [`ZIndex::Global`] is equivalent.
```

Comment:
nit: I'm not sure what "the equivalent" is since we didn't really define equivalence. What about:
```rust
/// For root UI nodes (nodes that have no parent), this is treated as [`ZIndex::Global`].
```

Reply:
Right, we should specify that it uses the same internal integer.

Reply:
Changed for a more complete description